### DISCUSSION TITLE
install leantime on subfolder but error 500

### DISCUSSION THREAD

**Discussion by nitiphone** [Leantime/leantime#619] [Q&A]
I am installing leantime on subfolder but got error 500
my step is create db, set config/configuration.php for bdb.com.la/dev/leantime
 after access bdb.com.la/dev/leantime/install I got error 500

Please kindly help me check?

**Comment by marcelfolaron** [comment]
What are the errors in resources/logs/errors log

**Reply by nitiphone** [reply]
[root@host logs]# cat bdb_com.la.php.error.log 
[07-Feb-2022 16:42:44 America/Los_Angeles] PHP Warning:  require_once(../vendor/autoload.php): failed to open stream: No such file or directory in /home/cp227590/public_html/bdb.com.la/dev/leantime/src/core/class.autoload.php on line 43
[07-Feb-2022 16:42:44 America/Los_Angeles] PHP Fatal error:  require_once(): Failed opening required '../vendor/autoload.php' (include_path='.:/opt/cpanel/ea-php73/root/usr/share/pear') in /home/cp227590/public_html/bdb.com.la/dev/leantime/src/core/class.autoload.php on line 43

**Reply by nitiphone** [reply]
now it can show Installation form and after click install, it just show the same page without any process. is there any logs?

**Comment by marcelfolaron** [comment]
Did you build Leantime from master or use the release version?

If from master you also need to run npm install and the grunt tasks. More
info at https://docs.leantime.io/#/

**Comment by marcelfolaron** [comment]
All logs are in resources/logs

Do you have mod_rewrite installed?

**Reply by nitiphone** [reply]
yes, I contact web hosting and they said it's default enable
what should I do?

**Comment by marcelfolaron** [comment]
Did you follow the instructions to make the changes to the graces files?

**Reply by nitiphone** [reply]
I follow these step for my live server
Download latest release package
Create an empty MySQL database
Upload entire directory to your server
Point your domain to the public/ directory
Rename config/configuration.sample.php to config/configuration.php
Fill in your database credentials (username, password, host, dbname) in config/configuration.php
Navigate to <yourdomain.com>/install
Follow instructions to install database and set up first user account

**Comment by marcelfolaron** [comment]
There are a few extra steps for subfolder installs as described here:

https://docs.leantime.io/#/installation/quick-installation?id=subfolder-installation

**Reply by nitiphone** [reply]
Now I reinstall the leantime on my subdomain to make sure it can support the PHP version. 
on install page, I can submit request to create the db
https://leantime.bdb.com.la/install
- it created database's table success.
- but when i go back to url https://leantime.bdb.com.la/ it's error 500
these are error message on resource message
[07-Feb-2022 19:18:45 America/Los_Angeles] PHP Deprecated:  Required parameter $userId follows optional parameter $projectId in /home/leantimebdbcom/public_html/src/domain/timesheets/repositories/class.timesheets.php on line 229
[07-Feb-2022 19:18:45 America/Los_Angeles] PHP Fatal error:  Uncaught TypeError: PDOStatement::setFetchMode(): Argument #2 must be a valid class in /home/leantimebdbcom/public_html/src/domain/reports/repositories/class.reports.php:169
Stack trace:
#0 /home/leantimebdbcom/public_html/src/domain/reports/repositories/class.reports.php(169): PDOStatement->setFetchMode(8, 'leantime\\domain...')
#1 /home/leantimebdbcom/public_html/src/domain/reports/services/class.reports.php(33): leantime\domain\repositories\reports->checkLastReportEntries('3')
#2 /home/leantimebdbcom/public_html/src/domain/dashboard/controllers/class.show.php(35): leantime\domain\services\reports->dailyIngestion()
#3 /home/leantimebdbcom/public_html/src/core/class.frontcontroller.php(148): leantime\domain\controllers\show->__construct()
#4 /home/leantimebdbcom/public_html/src/core/class.frontcontroller.php(107): leantime\core\frontcontroller->executeAction('dashboard.show')
#5 /home/leantimebdbcom/public_html/src/core/class.application.php(91): leantime\core\frontcontroller->run()
#6 /home/leantimebdbcom/public_html/public/index.php(41): leantime\core\application->start()
#7 {main}
  thrown in /home/leantimebdbcom/public_html/src/domain/reports/repositories/class.reports.php on line 169
[07-Feb-2022 19:18:47 America/Los_Angeles] PHP Deprecated:  Required parameter $userId follows optional parameter $projectId in /home/leantimebdbcom/public_html/src/domain/timesheets/repositories/class.timesheets.php on line 229
[07-Feb-2022 19:18:47 America/Los_Angeles] PHP Fatal error:  Uncaught TypeError: PDOStatement::setFetchMode(): Argument #2 must be a valid class in /home/leantimebdbcom/public_html/src/domain/reports/repositories/class.reports.php:169
Stack trace:
#0 /home/leantimebdbcom/public_html/src/domain/reports/repositories/class.reports.php(169): PDOStatement->setFetchMode(8, 'leantime\\domain...')
#1 /home/leantimebdbcom/public_html/src/domain/reports/services/class.reports.php(33): leantime\domain\repositories\reports->checkLastReportEntries('3')
#2 /home/leantimebdbcom/public_html/src/domain/dashboard/controllers/class.show.php(35): leantime\domain\services\reports->dailyIngestion()
#3 /home/leantimebdbcom/public_html/src/core/class.frontcontroller.php(148): leantime\domain\controllers\show->__construct()
#4 /home/leantimebdbcom/public_html/src/core/class.frontcontroller.php(107): leantime\core\frontcontroller->executeAction('dashboard.show')
#5 /home/leantimebdbcom/public_html/src/core/class.application.php(91): leantime\core\frontcontroller->run()
#6 /home/leantimebdbcom/public_html/public/index.php(41): leantime\core\application->start()
#7 {main}
  thrown in /home/leantimebdbcom/public_html/src/domain/reports/repositories/class.reports.php on line 169
